eZ/Bundle/PlatformInstallerBundle/src/Command/CheckUnsupportedPasswordHashTypesCommand.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformInstallerBundle\Command;
+
+use Doctrine\DBAL\Connection;
+use eZ\Publish\API\Repository\Values\User\User;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class CheckUnsupportedPasswordHashTypesCommand extends Command
+{
+    /** @var \Doctrine\DBAL\Connection */
+    private $connection;
+
+    public function __construct(
+        Connection $connection
+    ) {
+        $this->connection = $connection;
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this->setName('ezplatform:check-unsupported-password-hash-types');
```

Comment:
Naming. We already have `ezplatform:user` namespace, so we should continue to use it. Also let's try to make command name a bit shorter by being more general:
```suggestion
        $this->setName('ezplatform:user:validate-password-hashes');
```

Optionally maybe we should allow validation for a list of users (all if empty) so the solution is more flexible.
In this case:
```suggestion
        $this->setName('ezplatform:user:validate-password-hash');
```
with Input Argument of INPUT_ARRAY (or sth like that) type, so you can use it:
```
php bin/console ezplatform:user:validate-password-hash [user1] [user2]....
```
In both cases invalid hash should return non-zero result from the command